### DISCUSSION TITLE
Package alias to prevent naming conflict

### DISCUSSION THREAD

**Discussion by PedroHase** [keplerproject/luarocks#1480] [Q&A]
Hey all,

is it possible to force packages to be installed under an alias or somehow else prevent naming issues?

Reason I am asking is I had faced the issue that a dependency (`lua-http`) had used an OpenSSL package ( `luaossl`) I was unaware of while I wanted to use a different package (`lua-openssl`) and both packages used the same `openssl` name. This lead to the issue that `luaossl` took precedence and I could not use methods from `lua-openssl`.

Cheers, and happy new year ✨

**Comment by daurnimator** [comment]
FWIW I brought this conflict up a while ago: https://github.com/zhaozg/lua-openssl/issues/72
